backend/Application/Aggregates/Contract/Entities/Contract.cs

```diff
@@ -20,6 +20,7 @@ public sealed class Contract : BaseIdentification
     public ulong ContractAddressIndex { get; init; }
     public ulong ContractAddressSubIndex { get; init; }
     public string ContractAddress { get; init; } = null!;
+    [GraphQLIgnore]
```

Comment:
What does this do exactly and why is it part of this PR?

Reply:
I’m updating the module schema and to have it consistent I’m also updating other entities.

Reply:
Yes it removed the field from the graphql schema. Public properties are included as default and needs to be removed explicitly.